cluster-api/src/main/java/io/aiven/klaw/clusterapi/models/consumergroup/OffsetsTiming.java

```diff
@@ -0,0 +1,14 @@
+package io.aiven.klaw.clusterapi.models.consumergroup;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public enum OffsetsTiming {
+  BEFORE_OFFSET_RESET("BEFORE_OFFSET_RESET"),
```

Comment:
Do we need the _RESET here or is that a bit redundant?

Reply:
Was thinking of 'action', before and after